bin/clover/src/pipeline-steps/generateSubAssets.ts

```diff
@@ -88,6 +88,15 @@ export function generateSubAssets(incomingSpecs: PkgSpec[]): PkgSpec[] {
           schemaVariant.sockets.push(propInputSocket);
         }
 
+        // reuse existing assets so we don't recreate the same asset over and
+        // over again
+        const maybeExistingSubAsset = newSpecsByHash[hash];
+        if (maybeExistingSubAsset) {
+          logger.debug(`Existing subasset found: ${name}`);
+          maybeExistingSubAsset.names.push(name);
+          continue;
+        }
```

Comment:
any reason to move this lower? I feel like stopping it as early as possible is best

Reply:
because we weren't creating the input sockets on the parent prop by skipping where we were